todo/src/reducers/index.js

```diff
@@ -0,0 +1,39 @@
+import { ADD_TODO, TOGGLE_TODO, DELETE_TODO } from "../actions";
+
+
+const initialState = {
+  todos: [
+    { value: "Test todo", completed: false, id: 0 },
+    { value: "Another todo", completed: false, id: 1 }
```

Comment:
Good job adding an ID here makes it much more semantic and easier to manage when doing CRUD.